docs/getting-started/installing-desktop.rst

```diff
@@ -1,7 +1,25 @@
 .. include:: /stub-topic.txt
 
-|stub-icon| Installing .NET Framework
-=====================================
+Installing .NET Desktop Framework
```

Comment:
It's call ".NET Framework". "Desktop" is a colloquialism.

Do we need this documentation in the .NET Core docs?